types/artifacts.d.ts

```diff
@@ -673,6 +675,12 @@ declare global {
         observedSpeedIndex: number;
         observedSpeedIndexTs: number;
       }
+
+      export interface FormElement {
```

Comment:
I think this should be named `Form` (and the gatherer `forms.js`), and then the property `form` renamed to `attributes`. kinda odd to have it be `FormElement` but then have a property called `form`. wdyt?

Reply:
makes sense, let me edit